src/lbaf/Execution/lbsRuntime.py

```diff
@@ -121,7 +121,7 @@ def get_work_model(self):
         """Return runtime work model."""
         return self.__work_model
 
-    def execute(self, p_id: int, phase_increment: int=0):
+    def execute(self, p_id: int, phase_increment: int=0, lb_iterations=False):
```

Comment:
Flagging that this argument is not used in the function--is it necessary?

Reply:
Ah yes, but the plan is to use it in the future after discussion about the functionality with @lifflander 
Thanks for noticing.